custom-cakes/modules/footer/__container/footer__container.css

```diff
@@ -0,0 +1,67 @@
+.footer__container {
+  position: relative;
+}
+
+.footer__container::after {
```

Comment:
What is it?

Reply:
Oh. It's a separator line. Could we replace it with just a border-bottom of the previous container? It'll be more obvious

Reply:
Fixed.
I added aditional container (footer__bottom) for the social icons and copyright block. Also I duplicate the social list to the footer__top section to show it in the mobile and desktop versions.
In the footer__bottom I use the same social list for the tablet version.
Used media queries and display: none/block(or flex)